CHANGELOG.md

```diff
@@ -1,4 +1,47 @@
 ## 0.6.0 (Unreleased)
+
+**Implemented enhancements:**
+
```

Comment:
Add the following under "Implemented enhancements" to better highlight the enhancements the team made:

- New parameters about pagination for datasources [\#237](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/237) ([coderGo93](https://github.com/coderGo93))
- Added support for cluster autoscaling attributes [\#233](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/233) ([coderGo93](https://github.com/coderGo93))
- Migrated to new Terraform SDK [\#229](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/229) ([PacoDw](https://github.com/PacoDw))
- Added attribute container\_id to the cluster resource (useful for when a cluster exists before creating a peering connection) [\#208](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/208) ([coderGo93](https://github.com/coderGo93))
- Added attributes to snapshot restore jobs resource and datasources to support continuous cloud backup [\#224](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/224) ([coderGo93](https://github.com/coderGo93))
- General documentation improvements, Guide section add, and Upgrade Guide for 0.6.0 [\#240](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/240) ([themantissa](https://github.com/themantissa))
- General documentation improvements: connection string doc [\#223](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/223) ([themantissa](https://github.com/themantissa))
- General documentation improvements: network peering doc [\#217](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/217) ([themantissa](https://github.com/themantissa))

Reply:
done